_includes/scenarios/integrate/succeeding-attributes-to-update-their-values.md

```diff
@@ -0,0 +1,83 @@
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/079d7602-95af-4ddf-8fd5-cca5294038d9" id="Fv0-x9lenepV"></iframe></div>
+
+## Succeeding own RelationshipAttributes
+
+[RelationshipAttributes]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) are, as the name suggests, always associated with a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship).
+Thus, it is not possible to have unshared instances of them.
+Instead, you and your peer will always each have a LocalAttribute with the same RelationshipAttribute `content`.
+We call the LocalAttribute of the owner **own shared RelationshipAttribute** and the LocalAttribute of the peer **peer shared RelationshipAttribute**.
+
+### Creating and sharing a RelationshipAttribute
+
+Wanting to [create a RelationshipAttribute]({% link _docs_use-cases/use-case-consumption-create-and-share-a-relationshipattribute.md %}) always involves sharing it directly with a peer.
+To this end, a Request containing a [CreateAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#createattributerequestitem) with the RelationshipAttribute will be created and will be sent to the peer.
+Only if the peer accepts this Request, a LocalAttribute with the RelationshipAttribute as `content` will be created at their side and, by receiving the Response, at your side, too.
+Hence, you and your peer will always have an identical shared RelationshipAttribute, only differing in the LocalAttribute's `peer` property.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/11244175-05b4-4225-b08f-7da48aa21e2f" id="gw0-8Yqzp0IL"></iframe></div>
+
+### Succeeding a RelationshipAttribute and notifying the peer
+
+Equally to IdentityAttributes, again only the owner can [succeed a RelationshipAttribute]({% link _docs_use-cases/use-case-consumption-succeed-a-relationshipattribute-and-notify-peer.md %}).
+Doing so, a new version of the own shared RelationshipAttribute will be created at your side.
+The `succeeds` property of this LocalAttribute links to the old version, whose `succeededBy` property in turn is updated to the `id` of the newly created successor.
+Then, a Notification is sent automatically to the peer, which triggers the creation of a new peer shared RelationshipAttribute at their side, such that their LocalAttribute versions replicate the succession chain at your side.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/67c46978-696b-4adf-b04a-cf27d0438f53" id="ww0-.JinZtEX"></iframe></div>
```

Comment:
- I'd add a link to the "Succeed RelationshipAttribute and notify peer" use case for the corresponding box
- What I've noticed here: Maybe we should change the Address of the owner in the diagrams of all examples from "OwnAddress" to "OwnerAddress", because "OwnAddress" could seem like a placeholder (also, we are talking about owner, peer and "PeerAddress", so I think "OwnerAddress" could fit in that context)